course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -0,0 +1,101 @@
+### The `if` statement
+- the `if` statement is used to check if a conditional is `True`
+- syntax for writing the `if` statement:
+    1. use the `if` keyword at the beginning of the statement
+    2. follow the keyword with a correctly written conditional
+    3. finish the expression with colon 
+    4. block of code that will be executed if the statement passes - should be indented (usually by 4 spaces)
+
+An example of proper writing of the `if` statement:
+
+```py
+friend = "Rolf"
+name = input("Enter your name: ")
+if name == friend:
+    print(f"{name} is a friend")
+```
+
+Two examples of incorrect writing of the `if` statement:
+
+```py
+if name = friend:
+    print(f"{name} is a friend")
+```
+```py
+if name == friend:
+print(f"{name} is a friend")
+```
+**Extra resources**
+- [What is an if statement?](https://python.tecladocode.com/2_countries_visited/2_if_statements.html#what-is-an-if-statement)
+
+### The compound `if` statement - `elif` keyword
+- `elif` keyword is used for chaining multiple if statements
+- `elif` block will be entered only if the previous `if`/`elif` block doesn't get entered
+- the syntax is the same as with writing the `if` statement, the difference being that the `elif` keyword is used instead of `if`
+- `elif` keyword should be indented the same number of times as the previous `if`/`elif` keyword
```

Comment:
Above you said that the `elif` keyword is used for chaining multiple if statements. What if someone decides to use `if` instead of `elif`s like so:

```python
if <condition>:
    <statement>

if <condition>:
    <statement>

if <condition>:
    <statement>
```

What would be the difference between the `if` and `elif`? Then, someone can have nested `if` statements, and they won't be indented at the same level.

Reply:
The usage of the elif keyword is to chain if statements, the same would go for planes for example. The purpose of a plane is to skip the ground roads, but you can go road by road if you want - it doesn't change the plane's purpose :) 
I tried using an illustrative example here so I can break the dullness of our comments as well as make my point more fun :)